src/utils/logger.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+def get_logger (name: str, level = logging.DEBUG):
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.DEBUG)
```

Comment:
pretty sure you wanted to set the `level` here, instead of hard-coding logging.DEBUG (same below for `fh` and `ch`)